nox/sessions.py

```diff
@@ -242,6 +242,14 @@ def __init__(self, name, signature, func, global_config, manifest=None):
         self.manifest = manifest
         self.venv = None
 
+    @property
+    def description(self):
+        doc = self.func.__doc__
+        if doc:
+            first_line = doc.strip().split("\n")[0]
```

Comment:
Windows line endings a problem?

Reply:
I was thinking the same here, not sure about windows users, I'll try to do a little of research here

Reply:
Not completely sure, but looks like we are protected by https://www.python.org/dev/peps/pep-0278/. I don't have a windows O.S to test this in a _native_ way, but I have tested this forcing the line ending to be `dos` with `:set ff=dos` on vim and it works!